src/pages/SearchPage/Heatmap/Heatmap.style.js

```diff
@@ -0,0 +1,126 @@
+import styled from 'styled-components';
+
+const HeatmapWrapper = styled.section`
+  flex-grow: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  margin-top: 50px;
+`;
+
+const HeatmapContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 332px;
+  width: 1114px;
+  margin-top: 8px;
+`;
+
+const TopRow = styled.div`
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const HoursContainer = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  height: 100%;
+  width: auto;
+  color: #757575;
+  font-size: 14px;
+  font-weight: 500;
+  box-shadow: 0px -1px 2px -2px rgba(0,0,0,0.1);
+  &:first-child {
+    border-left: 1px solid #f3f3f3;
+  }
+  &:last-child {
+    border-right: 1px solid #f3f3f3;
+  }
+`;
+
+const Hour = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 53px;
+  width: 80px;
+  background: linear-gradient(180deg, #fefefe 0%, #e9e9e9 100%);
+  border-top: 1px solid #f3f3f3;
+`;
+
+const BottomRow = styled.div`
+  display: flex;
+  flex-direction: rows;
+  height: 280px;
+  width: 100%;
+`;
+
+const DaysContainer = styled.div`
+  width: calc(100% - 960px);
+  background: pink;
+  font-size: 0.875rem;
+  font-weight: 600;
+  letter-spacing: 0.8px;
+`;
+
+const Day = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  color: ${(props) => props.theme.colors.gray.light};
+  background: ${(props) => props.theme.heatmapStyle.day.background};
+`;
+
+const InfoContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(24, 1fr);
+  grid-template-rows: repeat(7, 1fr);
+  width: 960px;
+`;
+
+const InfoBlock = styled.button`
```

Comment:
I think it would be good to remove the outline here so the element doesn't seem to "glow" when focused